tmule/loader.py

```diff
@@ -14,13 +14,13 @@ def __init__(self, stream):
     def include(self, node):
 
         data = []
-        for file in str(self.construct_scalar(node)).split(' '):
+        for file in str(self.construct_scalar(node)).split(';'):
```

Comment:
This is is not backward-compatible change, and consequently also the CI test failed. We should try to avoid breaking existing configured configurations which this does if we are not careful. I think it’s now time to introduce versioning to our config files, so we can support different version of configurations files, e.g. the way https://docs.docker.com/compose/compose-file/compose-versioning/ does it: We should add an optional `Version` tag in the config file. I.e., if it is not present (old config) we assume `1`, i.e. the format before this change. With your proposed change we must then introduce version `2` for config files. Your new parser should only be used if a `version: 2` is present in the config file, otherwise the old semantics should be retained in my view. What does @adambinch think?

Reply:
We also have the options of:
- setting the divider based on the contents containing ";" (though this is hackish)
- defining this behaviour under a separate constructor
Of which the latter may offer the greatest backwards compatibility but may also lead to bloated functionality.

Reply:
Or we could go a slightly different direction and offer the functionality of including through a named field. 

```yaml
windows:
    - include: path/to/file1.yaml
    - include: path/to/file2.yaml
    
    - name: server_core
      panes: python command.py
```

As I undertsand it, this functionality is not currently possible. If we include any contents to `windows:` we must only include content.

Reply:
On further thought, any approach which makes use of this divider functionality is prone to failure if any path contains whatever the divider is set to (however unlikely that may be). I would put extra emphasis on the latter of my suggestions in that we use an explicit system of defining each file independently, then join their contents once loaded. Utilising this approach, we can keep the existing `!include` as it is for un-versioned files, and utilise a new yaml constructor which does not utilise the split mechanic for version2 files. I will be happy to put this together tomorrow if there is no objection to this formatting.